libs/RobotKit/include/RobotController.h

```diff
@@ -267,10 +281,18 @@ class RobotController : public interface::RobotController
 		}
 	}
 
+	void raiseAutonomousActivity() { raise(system::robot::sm::event::autonomous_activity_requested {}); }
```

Comment:
pareil ici : "raise autonomous activity" ça veut pas dire grand chose en anglais.

```suggestion
	void raiseAutonomousActivitiesModeRequested() { raise(system::robot::sm::event::autonomous_activity_requested {}); }
```